source/blog/2015-10-27-cappcon-2016.markdown

```diff
@@ -0,0 +1,23 @@
+---
+title: CappCon 2016
```

Comment:
CappCon is a bit cocky no? Let's see how many people are interested in the event first by calling that an "annual meet up" :)